### PR TITLE
Incremented version to 1.3.2.

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Incremented version to 1.3.2.

### Category
[Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation]

### Issues Resolved
part of https://github.com/opensearch-project/opensearch-build/issues/1882

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [x] New functionality includes testing
- [x] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]

**Comment by peternied** [comment]
Note; this change cannot be merged until there is a successful 13.2 distribution build with security for the integration tests to run against.

**Comment by cliu123** [comment]
The `https://ci.opensearch.org...` link gets access denied.

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ jobs:
     steps:
       - name: Download OpenSearch Core
         run: |
-          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.1/latest/linux/x64/builds/opensearch/dist/opensearch-min-1.3.1-linux-x64.tar.gz
+          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/builds/opensearch/dist/opensearch-min-1.3.2-linux-x64.tar.gz
```
Comment (cliu123) [inline .github/workflows/integration-test.yml:12]:
```suggestion
          wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-1.3.2-linux-x64.tar.gz
```

Comment (zelinh) [reply .github/workflows/integration-test.yml]: @cliu123 Thanks for the suggestions. I updated the changes in new commit.

**Comment by peternied** [comment]
@zelinh it looks like GitHub is requesting for the changes to be rebased before it can be merged.  Let me know if you'd like a hand with this.

**Pull request closed without merging**